discovery/triton/triton.go

```diff
@@ -102,11 +105,12 @@ func init() {
 // DiscoveryResponse models a JSON response from the Triton discovery.
 type DiscoveryResponse struct {
 	Containers []struct {
-		ServerUUID  string `json:"server_uuid"`
-		VMAlias     string `json:"vm_alias"`
-		VMBrand     string `json:"vm_brand"`
-		VMImageUUID string `json:"vm_image_uuid"`
-		VMUUID      string `json:"vm_uuid"`
+		Groups      []string `json:"groups:`
```

Comment:
The tag is off here

Reply:
I assume you're talking about the `:` at the end of `json:"groups:` and the missing `"`? If so it's amazing this actually works with it like that :). I'll fix this right now.

Reply:
Fixed